doc/chapters/getting-started/index.rst

```diff
@@ -0,0 +1,106 @@
+
+Getting started
+***************
+
+Structure
+=========
+
+The user API is implemented by the SoftwareContainer Agent as a D-bus service. That API is used
+to start, configure, and control containers. On the bus, the Agent has the name
+`com.pelagicore.SoftwareContainer` and object path `/com/pelagicore/SoftwareContainerAgent`.
+
+The Agent in turn links to the SoftwareContainer library which is to be considered an internal
+API for most users.
+
+TODO: Explain the thoughts around root privileges and the Agent.
+
+
+Container config
+================
+
+When creating containers, a configuration is passed as a JSON string. The string is an array with objects, in the JSON sense.
+The currently supported configs are:
+
+  * Disk writing buffer - on or off. Key: "enableWriteBuffer", value: `true` for on or `false` for off.
+
+Example config JSON::
+
+    [{
+        "enableWriteBuffer": true
+    }]
+
+The main reason this config is passed as raw JSON is to support additions in supported config options without breaking the API.
+
+
+Working with containers
+=======================
+
+Throughout this section, we will create a container, run an application in it and then shut down the
+container again.
+
+Prerequisites:
+  * configure and build (see project `README.md`)
+
+
+From the build directory, run the agent::
+
+    sudo ./agent/softwarecontainer-agent
+
+Not that we are running this command with `sudo`, the Agent needs to be started with root privileges.
+
+
+Using `dbus-send`, introspect the Agent D-Bus service API::
+
+    dbus-send --system --print-reply --dest=com.pelagicore.SoftwareContainerAgent /com/pelagicore/SoftwareContainerAgent org.freedesktop.DBus.Introspectable.Introspect
```

Comment:
I think you should split this over several lines to make it easier and faster to read.
Using '\' will persevere the "copy and paste"-ability if that is a concern.
Eg:
```bash
dbus-send --system --print-reply \
    --dest=com.pelagicore.SoftwareContainerAgent \
      /com/pelagicore/SoftwareContainerAgent \
      org.freedesktop.DBus.Introspectable.Introspect
```